extras/src/test/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactoryTest.java

```diff
@@ -167,6 +167,25 @@ public void testSerializeCollidingTypeFieldName() {
     }
   }
 
+  public void testSerializeWrappedNullValue() {
+    TypeAdapterFactory billingAdapter = RuntimeTypeAdapterFactory.of(BillingInstrument.class)
```

Comment:
Ahhh, yes, yes good point. For whatever reason I was thinking that this class had been imported into Gson internal.